uber/custom_tags.py

```diff
@@ -130,6 +130,19 @@ def urlencode(s):
 
 
 @JinjaEnv.jinja_filter
+def url_to_link(url=None, text=None, target=None):
+    # Jinja2 has a 'urlize' filter but it depends on links having `.com` or `http` in the name
+    # This works with relative links and allows you to also define link text
+    if not url:
+        return ''
+
+    if not text:
+        text = url
+
+    return safe_string('<a href="{}" {}>{}</a>'.format(url, 'target="{}"'.format(target) if target else '', text))
```

Comment:
HTML escaping was a great idea, but urlencoding turns out to not work, to my surprise. It seems that urlencoding is used to turn special characters like `/` into data for URL parameters. When you urlencode a regular link, like `/regular/link`, CherryPy returns a 404 error (though, amusingly, prints the URL correctly). See: https://stackoverflow.com/questions/37498904/link-doesnt-work-when-href-attribute-value-is-url-encoded